### ISSUE TITLE
Create and Test "Checkout Cart" Page and API

### ISSUE THREAD

**Issue by CoreyWarren** [CoreyWarren/coldcmerch.com#41]

**Comment by CoreyWarren** [comment]
Note: After some testing, it seems that Backend client secret setup and such must be configured before we can really test the front-end. This is because Stripe requires the client secret in order to do stuff like render the info input fields. (Sure, we can technically get CheckoutPage.js to display something, but let's focus on backend for now).

**Comment by CoreyWarren** [comment]
Closing this in favor of more specific issues for the checkout page. Congrats.
